packages/dd-trace/src/appsec/blocking.js

```diff
@@ -1,7 +1,11 @@
 'use strict'
 
 const fs = require('fs')
+
+// TODO: move template loading to a proper spot. See APPSEC-8135.
+let templateLoaded = false
 let templateHtml, templateJson
+
 function block (req, res, topSpan, abortController) {
```

Comment:
Also need to be renamed:
- a bit below in the same file
- 14 occurences in appsec/reporter.js
- 5 occurences in test/appsec/index.spec.js
```suggestion
function block (req, res, rootSpan, abortController) {
```

Reply:
I've renamed the other occurences of `topSpan` here in #2794